README.md

```diff
@@ -23,6 +23,8 @@ You should set these up before attempting to install the repo.
 
 1. Make sure your SOCKS access is running.
 
+1. Install vets-website in the same parent directory `https://github.com/department-of-veterans-affairs/vets-website`
```

Comment:
```suggestion
1. Clone the vets-website repo adjacent to next-build in the same parent directory.
    `git@github.com:department-of-veterans-affairs/vets-website.git`

1.  In vets-website, set yarn to the required version: `yarn set version 1.19.1`
1. `yarn install` and `yarn build`. You are now done in vets-website.

1. In the next-build directory, `yarn set version stable`

1. Run `yarn install`
```

Reply:
I am getting errors locally when installing vets-website, first because of the yarn version and now because of differences in M1 Mac architecture. not a blocker for this ticket because the symlink is working... but I think we want to provide a little more detail than "install vets-website here"

Reply:
good call, added